.circleci/config.yml

```diff
@@ -439,8 +439,7 @@ jobs:
           name: Precompile assets
           command: |
             cp config/examples/*.yml config/
-            bundle exec rake assets:precompile RAILS_ENV=test
-            bundle exec rake assets:precompile RAILS_ENV=production
+            bundle exec rake assets:precompile RAILS_ENV=test WEBPACKER_PRECOMPILE=false
```

Comment:
Why removing this?

Reply:
Apparently after some update, the rest of the tests were using the right packs (packs-test) so we don't need to compile both. Also, better to compile webpack assets with our custom `webpack:compile` task that doesn't use yarn